### ISSUE TITLE
Facing issue in balena local configure 

### ISSUE THREAD

**Issue by Sharvin26** [balena-io/balena-cli#1212]
I have downloaded the balena-cli from this [Link](https://github.com/balena-io/balena-cli/releases) I have unzipped the Balena image and set the balena-cli path using the following command:

`export PATH=$PATH:/usr/local/lib/balena-cli`

Now when I run 

`sudo balena local configure resin.img`

I get following error:

```
➜ sudo balena local configure resin.img
No such file or directory: /snapshot/balena-cli/node_modules/denymount/bin/denymount

Additional information may be available by setting a DEBUG=1 environment
variable: "set DEBUG=1" on a Windows command prompt, or "export DEBUG=1"
on Linux or macOS.

If you need help, don't hesitate in contacting our support forums at
https://forums.balena.io

For bug reports or feature requests, have a look at the GitHub issues or
create a new one at: https://github.com/balena-io/balena-cli/issues/
```

I have downloaded all the Requirements stated in this [documentation](https://github.com/balena-io/balena-cli/blob/master/INSTALL.md). 

```
➜ balena version
10.10.1
```

Note: I am using mac os for this process.

**Comment by pdcastro** [comment]
@Sharvin26, thank you for reporting this issue. I was able to reproduce it and I have a pull request that fixes it, so the fix should be released soon: https://github.com/balena-io/balena-cli/pull/1213

I believe this issue is also a duplicate of #977 [Getting Error Message No such file or directory: /snapshot/resin-cli/node_modules/denymount/bin/denymount](https://github.com/balena-io/balena-cli/issues/977)

**Comment by Sharvin26** [comment]
Okay.

**Comment by Sharvin26** [comment]
I am still facing the issue. Has the issue been fixed?

**Comment by pdcastro** [comment]
@Sharvin26, the pull request I linked is still open, as the reviewer requested some changes, and addressing those changes required the creation of a second pull request in another repository:
* https://github.com/balena-io/balena-cli/pull/1213
* https://github.com/balena-io-modules/denymount/pull/8

I am working on them at the moment. When the PRs are approved, they will be merged and the automated CLI release process will then produce a new CLI version. I will update issue #977 with the new CLI version number when it is available, and the issue will then be closed. You can watch that issue and/or the PRs for updates. Thanks again for reporting this issue.

**Comment by pdcastro** [comment]
@Sharvin26, this issue is fixed in CLI [v10.10.3 or later](https://github.com/balena-io/balena-cli/releases), released just now. If you have any problems, feel free to reopen it. Thanks again for reporting it.
